test/parallel_api/experimental/asynch.pass.cpp

```diff
@@ -30,6 +30,18 @@
 #if TEST_DPCPP_BACKEND_PRESENT
 #include "support/sycl_alloc_utils.h"
 
+class Copy1;
+class Fill1;
+class ForEach1;
+class Transform1; class Transform2;
+class Reduce1; class Reduce2; class Reduce3;
+class Scan;
+class Sort;
+class Copy2a; class Copy2b;
+class Scan2a; class Scan2b;
+class Fill2a; class Fill2b;
+class async1; class async2; class async3;
```

Comment:
I think this is another place where we can use a single class template due to the large number of names.